pkg/crc/config/viper_config_test.go

```diff
@@ -183,6 +184,10 @@ func TestViperConfigBindFlagSet(t *testing.T) {
 		Value:     6,
 		IsDefault: false,
 	}, config.Get(CPUs))
+
+	bin, err := ioutil.ReadFile(configFile)
+	assert.NoError(t, err)
+	assert.JSONEq(t, `{"cpus":6,"extra":"","nameservers":""}`, string(bin))
```

Comment:
Not clear if this is a 'theoritical' issue though, or if this can happen while using crc.